udata_front/theme/gouvfr/templates/macros/sort_search.html

```diff
@@ -0,0 +1,33 @@
+{% macro sort_search(result) %}
+{% if kwargs %}
+    {% set to_url = result.query.to_url %}
+    {% if in_url('sort') %}
+        {% if request.args.sort.startswith('-') %}
+            {% set current_sort = request.args.sort %}
+        {% endif %}
+    {% endif %}
+{% endif %}
+<div class="fr-grid-row align-items-center">
+    <label for="sort-search" class="fr-col-auto fr-text--sm fr-m-0 fr-mr-1w">
+        {{_('Sort by:')}}
+    </label>
+    <div class="fr-col">
+        <select
```

Comment:
I think the behaviour to change sort order is a bit weird without js. Indeed, we need to change the select, and trigger the search button above, that doesn't seem to be part of the same form. What do you think?

Reply:
Done